.github/workflows/e2e.yml

```diff
@@ -52,7 +52,24 @@ jobs:
           NODE_ENV: "production"
           CI_OS: ${{ runner.os }}
 
-      - name: Build Wrangler package for npm
+      - name: Pack Miniflare
+        run: pnpm pack --pack-destination $HOME
+        env:
+          NODE_ENV: "production"
+        working-directory: packages/miniflare
+
+      - name: Find Miniflare
+        shell: bash
+        id: "find-miniflare"
+        run: echo "dir=$(ls $HOME/miniflare-*.tgz)" >> $GITHUB_OUTPUT;
+
+      - name: Modify wrangler package.json miniflare dependency
+        run: cat <<< $(jq --tab ".dependencies.miniflare = \"$MINIFLARE_VERSION\"" packages/wrangler/package.json) > packages/wrangler/package.json
```

Comment:
does this need to include a `file:` protocol?

Reply:
I don't think so, but can't hurt—I'll include one